functional_tests.go

```diff
@@ -2889,120 +3378,129 @@ func testGetObjectReadSeekFunctionalV2() {
 	if err != nil {
-		logger().Fatal("Error:", err, bucketName)
+		failureLog(function, args, startTime, "", "MakeBucket failed", err).Fatal()
 	}
 
 	// Generate 33K of data.
 	var reader = getDataReader("datafile-33-kB", thirtyThreeKiB)
 	defer reader.Close()
 
 	objectName := randString(60, rand.NewSource(time.Now().UnixNano()), "")
+	args["objectName"] = objectName
+
 	buf, err := ioutil.ReadAll(reader)
 	if err != nil {
-		logger().Fatal("Error:", err, bucketName, objectName)
+		failureLog(function, args, startTime, "", "ReadAll failed", err).Fatal()
 	}
 
 	// Save the data.
 	n, err := c.PutObject(bucketName, objectName, bytes.NewReader(buf), "binary/octet-stream")
 	if err != nil {
-		logger().Fatal("Error:", err, bucketName, objectName)
+		failureLog(function, args, startTime, "", "PutObject failed", err).Fatal()
 	}
 
 	if n != int64(thirtyThreeKiB) {
-		logger().Fatalf("Error: number of bytes does not match, want %v, got %v\n", thirtyThreeKiB, n)
+		failureLog(function, args, startTime, "", "Number of bytes does not match, expected "+string(int64(thirtyThreeKiB))+" got "+string(n), err).Fatal()
 	}
 
 	// Read the data back
 	r, err := c.GetObject(bucketName, objectName)
 	if err != nil {
-		logger().Fatal("Error:", err, bucketName, objectName)
+		failureLog(function, args, startTime, "", "GetObject failed", err).Fatal()
 	}
 
 	st, err := r.Stat()
 	if err != nil {
-		logger().Fatal("Error:", err, bucketName, objectName)
+		failureLog(function, args, startTime, "", "Stat failed", err).Fatal()
 	}
+
 	if st.Size != int64(thirtyThreeKiB) {
-		logger().Fatalf("Error: number of bytes in stat does not match, want %v, got %v\n",
-			thirtyThreeKiB, st.Size)
+		failureLog(function, args, startTime, "", "Number of bytes in stat does not match, expected "+string(int64(thirtyThreeKiB))+" got "+string(st.Size), err).Fatal()
 	}
 
 	offset := int64(2048)
 	n, err = r.Seek(offset, 0)
 	if err != nil {
-		logger().Fatal("Error:", err, offset)
+		failureLog(function, args, startTime, "", "Seek failed", err).Fatal()
 	}
 	if n != offset {
-		logger().Fatalf("Error: number of bytes seeked does not match, want %v, got %v\n",
-			offset, n)
+		failureLog(function, args, startTime, "", "Number of seeked bytes does not match, expected "+string(offset)+" got "+string(n), err).Fatal()
 	}
 	n, err = r.Seek(0, 1)
 	if err != nil {
-		logger().Fatal("Error:", err)
+		failureLog(function, args, startTime, "", "Seek failed", err).Fatal()
 	}
 	if n != offset {
-		logger().Fatalf("Error: number of current seek does not match, want %v, got %v\n",
-			offset, n)
+		failureLog(function, args, startTime, "", "Number of seeked bytes does not match, expected "+string(offset)+" got "+string(n), err).Fatal()
 	}
 	_, err = r.Seek(offset, 2)
 	if err == nil {
-		logger().Fatal("Error: seek on positive offset for whence '2' should error out")
+		failureLog(function, args, startTime, "", "Seek on positive offset for whence '2' should error out", err).Fatal()
```

Comment:
No point of logging nill error message, err:
failureLog(function, args, startTime, "", "Seek on positive offset for whence '2' should error out"~~, err~~).Fatal()